### PR TITLE
Alcohol percentages!

### PR DESCRIPTION
**Pull request by FlattestGuitar**
Okay, so I might have totally botched this, please don't shout.

This changes the alcohol system a bit and makes booze have defined levels of ethanol. 

As it stands all the `***_start` values are roughly twice the number of units of whiskey you'd need to reach that level of intoxication, whiskey being a 40% beverage.

Generally all the exotic drinks using plasma and other weird ingredients have more ethanol than they technically should have.

:cl: FlattestGuitar
tweak: Alcoholic beverages now have proper levels of ethanol in them. Good luck passing out after a beer.
/:cl:

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
This is going to make it practically impossible to get drunk.

Beer? You'll have to absorb over <b><u>400</u></b> units before you even start *slurring*.

**Comment by FlattestGuitar** [comment]
The number's actually 80u, so just over 1.5 glasses. Considering in real life it's hard to start slurring even after four to five glasses, I'd say that's reasonable.

**Comment by FlattestGuitar** [comment]
Alright, I fucked up. The weird old system is still there and I based my amounts off of it. Right now `d` doubles the beverage it's called by, making 20u of whiskey count as 40u. I'll fix this bit and reopen this.

Also, 1u of a beverage counts as 2.5 cycles, so it's much easier to get drunk on everything even not considering the woopsie.

**Comment by FlattestGuitar** [comment]
Alright, fixed it. 

As it stands, in order to start slurring it takes 20u of whiskey, 80u of beer or 8u of pure ethanol.

**Comment by Fox-McCloud** [comment]
If you're wanting to impact how long it takes you to get drunk, you don't need a bunch of changes like this to do it; just change the start number vars for the various effects on ethanol and be done with it; this is a lot easier to adjust for, for future alcohol types than having to adjust for that AND the new var you created here, too.

**Comment by FlattestGuitar** [comment]
But this makes it so that you have just one variable per drink to adjust and it scales properly instead of having to modify six different ones for every single drink.

I removed all the `***_start` variables on all children of ethanol that had them, because they didn't make much sense with the new percentage variable in place. Perhaps I should remove them alltogether and just type the numbers under the `on_mob_life` proc? That would clean things up quite a bit.

**Comment by Fox-McCloud** [comment]
Well, for the most part here, the problem with alcohol hasn't been that *individual* drinks are too weak or too strong, it's that a lot feel that everything is too strong.

This occurred because of some really shoddy coding with how alcohol was processed. Basically alcohol metabolized three times every time it processed (meaning 1.2 units would get removed as opposed to 0.4).

Tripling the start vars would put it back roughly where it was.

File: code/modules/reagents/oldchem/reagents/drink/reagents_alcohol.dm [hunk 1]
```diff
@@ -11,31 +11,29 @@
...
-	var/confused_start = 130	//amount absorbed after which mob starts confusing directions
-	var/vomit_start = 180	//amount absorbed after which mob starts vomitting
-	var/blur_start = 260	//amount absorbed after which mob starts getting blurred vision
-	var/pass_out = 325	//amount absorbed after which mob starts passing out
+	var/slur_start = 20			//amount absorbed after which mob starts slurring
+	var/brawl_start = 25		//amount absorbed after which mob switches to drunken brawling as a fighting style
+	var/confused_start = 30	//amount absorbed after which mob starts confusing directions
+	var/vomit_start = 45 //amount absorbed after which mob starts vomitting
+	var/blur_start = 70	//amount absorbed after which mob starts getting blurred vision
+	var/pass_out = 90	//amount absorbed after which mob starts passing out
```
Comment (Fox-McCloud) [inline code/modules/reagents/oldchem/reagents/drink/reagents_alcohol.dm]: These are the vars I'm referring to that could most easily be adjusted.

Comment (FlattestGuitar) [inline code/modules/reagents/oldchem/reagents/drink/reagents_alcohol.dm]: I realise they can be adjusted, but it seems a bit too messy to adjust all six of them for every single type of alcohol. I still don't like the system of a unit of beer getting you to the same level of drunkenness as a unit of whiskey. Those two are different and people should drink them differently.

What people said in [this](http://nanotrasen.se/phpBB3/viewtopic.php?f=12&t=7422) thread was that the drunkeness depended on tha number of "drinks" someone had. The amount of the beverage per drink should depend on its type. Making vodka more powerful than cider seems more sensible to me.

**Comment by FlattestGuitar** [comment]
Okay, well, I'd need some pointers here. 

I think the options right now are to either adjust all the `***_start` variables for different alcohols to make them have a different effect OR remove the variables alltogether keeping just `alcohol_perc` and typing in the right numbers under the `on_mob_life` proc. 

Which one seems better, @Fox-McCloud?

**Comment by Fox-McCloud** [comment]
Still don't like this.

**Pull request merged**